stripe-ui-core/src/test/java/com/stripe/android/uicore/utils/README.md

```diff
@@ -1,4 +1,4 @@
-# PaymentSheet screenshot tests
+# stripe-ui-core screenshot tests
```

Comment:
Instead of moving it, can you duplicate it for now? We can then move it into a future `ui-testing` module.

Reply:
done